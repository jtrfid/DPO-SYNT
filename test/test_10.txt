5 3

0 m
1 m
2 m
3 m
4 m

a c u
b c o
c u u

a 1
b 3 c 2
a 0 b 4
c 0 b 4
c 2