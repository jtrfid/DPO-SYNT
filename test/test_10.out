FSM: 
12 3

(0,0,0) m
(1,1,1) m
(3,3,2) m
(2,2,dead) m
(4,dead,dead) m
(0,0,dead) m
(2,dead,dead) m
(1,1,dead) m
(0,dead,dead) m
(3,3,dead) m
(1,dead,dead) m
(3,dead,dead) m

a c u
b c o
c u u

a (1,1,1)
c (2,2,dead) b (3,3,2)
b (4,dead,dead) c (0,0,0)
b (4,dead,dead) a (0,0,dead)
c (2,dead,dead)
a (1,1,dead)
b (4,dead,dead) a (0,dead,dead)
c (2,2,dead) b (3,3,dead)
a (1,dead,dead)
b (4,dead,dead) c (0,0,dead)
c (2,dead,dead) b (3,dead,dead)
b (4,dead,dead) c (0,dead,dead)
REQ_FSM: 
3 3

(0,0,0) m
(1,1,1) m
(3,3,2) m

a c u
b c o
c u u

a (1,1,1)
b (3,3,2)
c (0,0,0)
*******************************************************************
*********************All Inclusive Controller**********************
*******************************************************************
Y-States:
	Information State: {(0,0,0)}
	Transitions to:
		Z-State with Information State {{(0,0,0),(1,1,1),(2,2,dead),(0,0,dead),(1,1,dead)},{a}} via Control Decision {a}
		Z-State with Information State {{(0,0,0)},{}} via Control Decision {}
Z-States:
	Information State: {{(0,0,0)},{}}
	Information State: {{(0,0,0),(1,1,1),(2,2,dead),(0,0,dead),(1,1,dead)},{a}}
*******************************************************************
*********************End All Inclusive Controller******************
*******************************************************************
No Solution
Press any key to continue...
